docs/development/new-package-manager-template.md

```diff
@@ -74,14 +74,6 @@
 - [ ] Supports range constraints (e.g `^1.0.0` or `1.x`)
 - [ ] No
 
-### Is this package manager used for applications, libraries, or both? If both, is there a way to tell which is which?
```

Comment:
https://github.com/renovatebot/renovate/blob/a762677d6abafe60758e220ca26d73d0c10acfac/docs/usage/configuration-options.md?plain=1#L2298-L2329

Maybe you need to change things in these docs as well?